custom/intrahealth/reports/fiche_consommation_report.py

```diff
@@ -19,7 +18,20 @@ class FicheConsommationReport(IntraHealtMixin, DatespanMixin, GenericTabularRepo
 
     @property
     def model(self):
-        return FicheData(config=self.report_config)
+        config = self.report_config
+        locations = []
+        if 'region_id' in config:
+            locations = tuple(SQLLocation.objects.get(
+                location_id=config['region_id']
+            ).archived_descendants().values_list('location_id', flat=True))
+        elif 'district_id' in config:
+            locations = tuple(SQLLocation.objects.get(
+                location_id=config['district_id']
+            ).archived_descendants().values_list('location_id', flat=True))
+
+        if locations:
+            config.update({'archived_locations': locations})
```

Comment:
not blocking but i was confused by the variable name until I saw how it was getting used. might be better to use `archived_locations`